echtvar/0.1.9/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM staphb/bcftools:1.18
```

Comment:
Might be good to also add the label maintainer and description

Reply:
Done